tests/func/test_version.py

```diff
@@ -0,0 +1,20 @@
+from dvc.main import main
+from dvc.command.version import CmdVersion
+from dvc.cli import parse_args
+
+from tests.basic_env import TestDvc
+
+
+class TestVersion(TestDvc):
+    def test_run(self):
+        ret = main(["version"])
+        self.assertEqual(ret, 0)
+
+
+def test_info(dvc, mocker, caplog):
+    cmd = CmdVersion(parse_args(["version"]))
+    cmd.run()
+    for record in caplog.records:
```

Comment:
Maybe you might want to mock the `__version__` and `platform`, just to make sure that you are indeed calling those.

Or extend the assertion to a regex: "DVC version: \d+` (or something that asserts that it is returning the whole message.

Reply:
@mroutis Okay, I will write a suitable regex pattern to capture the log properly.